flink-connector-influxdb2/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,31 @@
+#
```

Comment:
Please remove this file entirely. It can easily overwrite the user's configuration during shading and confuse the heck out of them.